src/components/molecules/navSearch/NavSearchDropdown.tsx

```diff
@@ -49,34 +49,38 @@ export const NavSearchDropdown = ({ contextualSearchName, isEverything, setIsEve
     },
   ];
   if (!isEverything) dropdownOptions.reverse();
-  const [firstOption, secondOption] = dropdownOptions;
 
   const handleClick = (newValue: boolean) => {
     setIsOpen((current) => !current);
     if (isOpen) setIsEverything(newValue);
   };
 
-  const optionClasses = "h-full w-full flex items-center justify-between gap-2 pl-3 pr-3 bg-surface-ui text-sm leading-4 text-nowrap cursor-pointer";
-
   return (
-    <div ref={ref} className="border-l border-l-border-light">
-      <button
-        type="button"
-        onClick={() => handleClick(firstOption.newIsEverythingValue)}
-        className={`${optionClasses} ${isOpen ? "rounded-tr-md" : "rounded-r-md"}`}
-      >
-        {firstOption.name}
-        <div className={isOpen ? "invisible" : ""}>
-          <LuChevronsUpDown height="12" width="12" />
-        </div>
-      </button>
-      <button
-        type="button"
-        onClick={() => handleClick(secondOption.newIsEverythingValue)}
-        className={`${optionClasses} ${isOpen ? "" : "invisible"} rounded-b-md`}
+    <div className="h-[40px] overflow-visible">
```

Comment:
Musing: this is one of those things that I don't love about our currently implementation of tailwind, we have the height specified here as well as in the `NavSearch` in the input. Obviously someone would quickly see they have missed one or other but longer term it would be nice to align this _kind of thing_, because I imagine we'll run into it in other places. No opinion on how yet, but just thinking aloud.

Reply:
Agreed, this was a compromise of needing a hard lock on the component height so that the dropdown and the input are the exact same height.